src/com/walmartlabs/lacinia/pedestal/subscriptions.clj

```diff
@@ -367,17 +369,16 @@
   :app-context
   : The base application context provided to Lacinia when executing a query.
 
-  :interceptors-configurator (default: identity)
-  : A function that is passed the map of interceptors and can modify the map before it is ordered
-    into a seq of interceptors."
+  :subscription-interceptors
+  : A seq of interceptors for processing queries.  The default is
+    derived from [[default-interceptors]]."
   [compiled-schema options]
```

Comment:
Fair enough. I can add a test with an extra interceptor that "proves" the provided :subscription-interceptors is actually used.